.github/workflows/test_production_build.yml

```diff
@@ -0,0 +1,78 @@
+name: Test production build
+
+on: push
+
+env:
+  APP_NAME: rails_templates
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
```

Comment:
Do we need to define the `DOCKER_IMAGE` and `DOCKER_REGISTRY_HOST` here @hoangmirs 🤔 do we use it somewhere?

Reply:
We don't need them here, they could be loaded from the env file, but I just want to make them consistent between workflows

Reply:
🤔  as we don't use it I think we should remove it to avoid confusion for the other contributor, also as you mentioned that `want to make them consistent` so how about the other env like

```
DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
```

As it's not in this workflow yet